src/main/java/pro/sky/courseworktelegrambot/entities/Report.java

```diff
@@ -0,0 +1,90 @@
+package pro.sky.courseworktelegrambot.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Objects;
+/**
+ * Object Report
+ */
+@MappedSuperclass
+public abstract class Report {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private LocalDate date; // дата отчета
+    @Lob
+    private byte[] photo; // фото отчета
+    @Lob
```

Comment:
```suggestion
    @Lob(colummnDefinition = "oid")
```
чтобы не хранить массив байт в ячейке, а хранить только object id на large object